### PR TITLE
chore: Improve explanations in ok pkg install --help

### PR DESCRIPTION
**Pull request by yngvark**
# Motivation

Be more precise and clear about what actually happens when running e.g.g `ok pkg install someFolder`.

### CODE REVIEW

File: cmd/pkg/install.go [hunk 1]
```diff
@@ -6,8 +6,18 @@ import (
 )
 
 var InstallCommand = &cobra.Command{
-	Use:           "install [stackName1, stackName2, ...]",
-	Short:         "Install or update Boilerplate packages",
+	Use:   "install [outputFolder ...]",
+	Short: "Install or update Boilerplate packages",
```
Comment (staticaland) [inline cmd/pkg/install.go:10]:
```suggestion
	Short: "Install or update Boilerplate packages.",
```

File: cmd/pkg/install.go [hunk 2]
```diff
@@ -6,8 +6,18 @@ import (
...
+	Long: `Install or update Boilerplate packages.
+
+If no arguments are used, the command installs all the packages specified in the package manifest file.
+
+If one or more output folders are specified, the command installs only the packages from the package manifest that has
+OutputFolder that matches one of the specified output folders.
+`,
+	Example: `ok install networking
```
Comment (staticaland) [inline cmd/pkg/install.go:18]:
```suggestion
	Examples: `ok install networking
```

File: cmd/pkg/install.go [hunk 3]
```diff
@@ -6,8 +6,18 @@ import (
...
 var InstallCommand = &cobra.Command{
-	Use:           "install [stackName1, stackName2, ...]",
-	Short:         "Install or update Boilerplate packages",
+	Use:   "install [outputFolder ...]",
+	Short: "Install or update Boilerplate packages",
+	Long: `Install or update Boilerplate packages.
+
+If no arguments are used, the command installs all the packages specified in the package manifest file.
+
+If one or more output folders are specified, the command installs only the packages from the package manifest that has
```
Comment (staticaland) [inline cmd/pkg/install.go:15]:
```suggestion
If one or more output folders are specified, the command installs only the packages from the package manifest that has
```

Tung setning 🙈

File: cmd/pkg/install.go [hunk 4]
```diff
@@ -6,8 +6,18 @@ import (
...
+OutputFolder that matches one of the specified output folders.
```
Comment (staticaland) [inline cmd/pkg/install.go:16]:
```suggestion
If one or more output folders are specified, the command installs only the packages from the package manifest that has
OutputFolder that matches one of the specified output folders.
```

Tung setning 🙈

Comment (staticaland) [reply cmd/pkg/install.go:16]: > When output folders are specified, the command installs only the packages from the package manifest whose OutputFolder matches one of the specified folders.

Comment (yngvark) [reply cmd/pkg/install.go:16]: Jeg vet, klarte ikke bedre. Kan prøve igjen.

**Pull request merged**